packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx

```diff
@@ -206,7 +205,13 @@ const DaoSettings: React.FC<DaoSettingFormProps> = ({ onClickBack }) => {
   }, [daoDetails, provider]);
 
   const callSaveSetting = async (formData: FormInputs) => {
-    console.log('formData', formData);
+    if (!isConnected) {
```

Comment:
same thing here. having a button disabled should be enough when it's not connected. no need to do this check IMO.